keepassxc-browser/css/keepassxc-browser.css

```
@@ -0,0 +1,14 @@
+.cip-genpw-icon {
+    position: absolute;
+    cursor: pointer;
+}
+
+.cip-genpw-icon.key {
+    background: url('chrome-extension://__MSG_@@extension_id__/icons/key.png') right no-repeat;
```

```suggestion
    background: url('./icons/key.png') right no-repeat;
```

Why not use a relative path? Should not this work?
Also this obviously leads to the fact that you need to duplicate the work here…